ui/app/AppLayouts/Chat/views/ContactsColumnView.qml

```diff
@@ -80,6 +80,8 @@ Item {
             icon.name: "search"
             MouseArea {
                 anchors.fill: parent
+                hoverEnabled: true
+                cursorShape: Qt.PointingHandCursor
```

Comment:
Shouldn't this be `cursorShape: Qt.IBeamCursor` like in the [original implementation](https://github.com/status-im/StatusQ/blob/41f67b59e954550207fb8882a5b58dd705cc4a38/src/StatusQ/Controls/StatusBaseInput.qml#L113).

I'm wondering why the existing one doesn't work. It might be that it is shadowed by the current `MouseArea`. Maybe triggering the AppSearch by the focus scope might help?

Reply:
@stefandunca it would make sense to set it to `Qt.IBeamCursor` if that `StatusBaseInput` is really an input field, but in this case it's just kind of a button as I see which looks like an input field, but it triggers popup. So there is no way to enter anything into that input field that's why I set it to `Qt.PointingHandCursor`.

Reply:
I didn't check it, though it is the search edit box directly. Thanks for clarifying.